pepper-apis/src/main/java/org/broadinstitute/ddp/content/I18nContentRenderer.java

```diff
@@ -289,7 +289,15 @@ public String renderToString(String template, Map<String, Object> context) {
         VelocityContext ctx = new VelocityContext(context);
         StringWriter writer = new StringWriter();
         engine.evaluate(ctx, writer, TEMPLATE_NAME, template);
-        return writer.toString();
+        String result = writer.toString();
+        if (result.contains("$")) {
```

Comment:
Can you give an example, or even better, write a test case?

Reply:
Sure. Test provided.